Documentation/gettingstarted/kube-router.rst

```diff
@@ -23,7 +23,7 @@ Download the kube-router DaemonSet template:
 
 .. code:: bash
 
-    curl -LO https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.4.0/daemonset/generic-kuberouter-only-advertise-routes.yaml
+    curl -LO https://raw.githubusercontent.com/cloudnativelabs/kube-router/master/daemonset/generic-kuberouter-only-advertise-routes.yaml
```

Comment:
Thanks for the suggestion, contrary to the proposal in the GH issue, we should not put `master` in here as we face the risk of this file to stop exist or breaking the integration with us knowing about. It would be best to pin to the most recent commit, assuming it the integration still works of course.

Reply:
Thanks for pointing this out, I'll point the URL to the latest release instead of master.